crates/typst/src/layout/grid/mod.rs

```diff
@@ -185,9 +185,9 @@ pub struct GridElem {
 
     /// How to fill the cells.
     ///
-    /// This can be a color or a function that returns a color. The function is
-    /// passed the cells' column and row index, starting at zero. This can be
-    /// used to implement striped grids.
+    /// This can be a color or a function that returns a color. The function
+    /// receives the cells' column and row index, starting from zero. This can
+    /// be used to implement striped grids.
```

Comment:
I think the word `index` should be plural here, maybe we can fix this in this PR.

```suggestion
    /// receives the cells' column and row indices, starting from zero. This
    /// can be used to implement striped grids.
```

Reply:
So you changed your mind after all. I wanted to change that, but thought that it can be either way.